### PR TITLE
replaced various abbreviations

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-6-0/platform-devices-10-6-0-bundle/10-6-0-21-23.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Issue</th>
 <tr>
 <td>
 Administration</td>
-<td > In the <b>Inventory</b> tab, a user without relevant permission will no longer see the inventory tree. Moreover, the error message being displayed if a user doesn´t have relevant permissions now shows the correct information. In the <b>Application access</b> tab, a user without relevant permission will no longer be able to edit.</td>
+<td > In the <b>Inventory</b> tab, a user without relevant permission will no longer see the inventory tree. Moreover, the error message being displayed if a user doesn´t have relevant permissions now shows the correct information.In the <b>Application access</b> tab, a user without relevant permission will no longer be able to edit.</td>
```
Comment (rssag) [inline content/release-10-6-0/platform-devices-10-6-0-bundle/10-6-0-21-23.md:51]: there's an unusual character sequence in this line. looks like there should only be a space character?

File: content/release-10-7-0/application-10-7-0-bundle/10-7-0-24.md [hunk 2]
```diff
@@ -250,7 +250,7 @@ MTM-31421</td>
 <tr>
 <td>
 Cockpit</td>
-<td > The "Map" widget now does not display more devices than configured as default pageSize for the map ( e.g. 40).
+<td > The "Map" widget now does not display more devices than configured as default pageSize for the map ( for example, 40).
```
Comment (rssag) [inline content/release-10-7-0/application-10-7-0-bundle/10-7-0-24.md:253]: is the space character after the opening bracket necessary?

**Review by rssag — APPROVED** [review]
approved, but there are two small suggestions/questions

File: content/release-10-10-0/applications-10-10-0-bundle/10-10-0-8.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ MTM-38172</td>
 <tr>
 <td>
 Export</td>
-<td > Measurements added from data points are now escaped with bracket notation in order to support special characters (e.g. whitespace) in fragment and series names.</td>
+<td > Measurements added from data points are now escaped with bracket notation in order to support special characters (such as whitespace) in fragment and series names.</td>
```
Comment (MWindel) [inline content/release-10-10-0/applications-10-10-0-bundle/10-10-0-8.md:188]: I would recommend to use "(for example, whitespace)" (also below) as that is also used in the rest of documentation. "Such as" is used without parentheses.

File: content/release-10-10-0/platform-10-10-0-bundle/10-10-0-8.md [hunk 4]
```diff
@@ -138,7 +138,7 @@ MTM-37441</td>
 <tr>
 <td>
 Administration</td>
-<td > Tenant domains used to allow underscore signs (_). However, those characters are not allowed in wildcard certificates (e.g. subdomain.cumulocity.com). Now underscores are no longer allowed in tenant domains.</td>
+<td > Tenant domains used to allow underscore signs (_). However, those characters are not allowed in wildcard certificates (such as subdomain.cumulocity.com). Now underscores are no longer allowed in tenant domains.</td>
```
Comment (MWindel) [inline content/release-10-10-0/platform-10-10-0-bundle/10-10-0-8.md:141]: see above

File: content/release-10-11-0/applications-10-11-0-bundle/10-11-0-4.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ MTM-34978</td>
 <tr>
 <td>
 Web SDK</td>
-<td > A new utility function <code>toObservable</code> has been added to the Web SDK as part of the @c8y/ngx-components library. It converts any value provided as an argument to an observable, that emits this value and then completes. Developers can use it as a convenient method to handle any value as an observable rather than mixing observables and other types (e.g. Promises), and eliminate any checks and conversions in their logic. </td>
+<td > A new utility function <code>toObservable</code> has been added to the Web SDK as part of the @c8y/ngx-components library. It converts any value provided as an argument to an observable, that emits this value and then completes. Developers can use it as a convenient method to handle any value as an observable rather than mixing observables and other types (such as Promises), and eliminate any checks and conversions in their logic. </td>
```
Comment (MWindel) [inline content/release-10-11-0/applications-10-11-0-bundle/10-11-0-4.md:73]: see above

File: content/release-10-6-0/platform-devices-10-6-0-bundle/10-5-8-12.md [hunk 6]
```diff
@@ -93,25 +93,25 @@ Basically, the more data you got in one notification package from the server, th
...
 <td style="text-align:left">UI</td>
-<td style="text-align:left">Tooltips for domain and tenant name fields have been updated to indicate that using underscore is not recommended. 
+<td style="text-align:left">Tooltips for domain and tenant name fields have been updated to indicate that using underscore is not recommended.
 </td>
 <td>MTM-29753</td>
 </tr>
 <tr>
 <td style="text-align:left">UI</td>
-<td style="text-align:left">The default “pageSize” parameter value has been reduced from 100 to 20 to improve load performance of certain views that display asset hierarchies (e.g. the navigator in the Cockpit application or the inventory role assignment page). 
+<td style="text-align:left">The default “pageSize” parameter value has been reduced from 100 to 20 to improve load performance of certain views that display asset hierarchies (such as the navigator in the Cockpit application or the inventory role assignment page).
```
Comment (MWindel) [inline content/release-10-6-0/platform-devices-10-6-0-bundle/10-5-8-12.md:108]: see above

**Review by MWindel — APPROVED** [review]

File: content/release-10-10-0/applications-10-10-0-bundle/10-10-0-8.md (hunk 3 above)
Comment (BeateRixen) [reply content/release-10-10-0/applications-10-10-0-bundle/10-10-0-8.md]: fixed

File: content/release-10-10-0/platform-10-10-0-bundle/10-10-0-8.md (hunk 4 above)
Comment (BeateRixen) [reply content/release-10-10-0/platform-10-10-0-bundle/10-10-0-8.md]: Fixed

File: content/release-10-11-0/applications-10-11-0-bundle/10-11-0-4.md (hunk 5 above)
Comment (BeateRixen) [reply content/release-10-11-0/applications-10-11-0-bundle/10-11-0-4.md]: Fixed

File: content/release-10-6-0/platform-devices-10-6-0-bundle/10-5-8-12.md (hunk 6 above)
Comment (BeateRixen) [reply content/release-10-6-0/platform-devices-10-6-0-bundle/10-5-8-12.md]: fixed

File: content/release-10-7-0/application-10-7-0-bundle/10-7-0-24.md (hunk 2 above)
Comment (BeateRixen) [reply content/release-10-7-0/application-10-7-0-bundle/10-7-0-24.md]: No, good catch! Fixed

**Pull request merged**